README.md

```diff
@@ -8,13 +8,15 @@ This is the official AiiDA plugin for [Quantum ESPRESSO](https://www.quantum-esp
+| Plugin | AiiDA | Python | Quantum ESPRESSO |
+|-|-|-|-|
+| `v3.4.0 < v4.0.0` | <img class="svg-badge" title="Compatible with aiida-core >=1.4.4,<2.0.0" src="https://img.shields.io/badge/AiiDA->=1.4.4,<2.0.0-007ec6.svg?logo=[aiida-logo-base64]"> |  [![PyPI pyversions](https://img.shields.io/pypi/pyversions/aiida-quantumespresso.svg)](https://pypi.org/project/aiida-quantumespresso.svg) | 6.0, 6.1, 6.2, 6.3, 6.4, 6.5, 6.6, 6.7 |
+| `v3.3.0 < v3.4.0` | <img class="svg-badge" title="Compatible with aiida-core >=1.4.4,<2.0.0" src="https://img.shields.io/badge/AiiDA->=1.4.4,<2.0.0-007ec6.svg?logo="> |  [![PyPI pyversions](https://img.shields.io/pypi/pyversions/aiida-quantumespresso.svg)](https://pypi.org/project/aiida-quantumespresso.svg) | 6.0, 6.1, 6.2, 6.3, 6.4, 6.5, 6.6, 6.7 |
+| `v3.1.0 < v3.3.0` | <img class="svg-badge" title="Compatible with aiida-core >=1.2.0,<2.0.0" src="https://img.shields.io/badge/AiiDA->=1.2.0,<2.0.0-007ec6.svg?logo=[aiida-logo-base64]"> |  [![PyPI pyversions](https://img.shields.io/pypi/pyversions/aiida-quantumespresso.svg)](https://pypi.org/project/aiida-quantumespresso.svg) | 6.0, 6.1, 6.2, 6.3, 6.4, 6.5, 6.6, 6.7 |
+| `v3.0.0 < v3.1.0` | <img class="svg-badge" title="Compatible with aiida-core >=1.0.0,<1.1.0" src="https://img.shields.io/badge/AiiDA->=1.0.0,<1.1.0-007ec6.svg?logo=[aiida-logo-base64]"> |  [![PyPI pyversions](https://img.shields.io/pypi/pyversions/aiida-quantumespresso/3.0.0.svg)](https://pypi.org/project/aiida-quantumespresso/3.0.0/) | 6.0, 6.1, 6.2, 6.3, 6.4, 6.5, 6.6, 6.7 |
+| `v2.0.0 < v3.0.0` | <img class="svg-badge" title="Compatible with aiida-core >=0.12,<1.0.0" src="https://img.shields.io/badge/AiiDA->=0.12,<1.0.0-007ec6.svg?logo=[aiida-logo-base64]"> |  [![PyPI pyversions](https://img.shields.io/pypi/pyversions/aiida-quantumespresso/2.1.svg)](https://pypi.python.org/pypi/aiida-quantumespresso/2.1.0/) | 6.0, 6.1, 6.2, 6.3, 6.4, 6.5, 6.6, 6.7 |
+
+Starting from `aiida-quantumespresso==4.0`, the last three minor versions of Quantum ESPRESSO are supported up to a maximum of two years.
```

Comment:
This might be interpreted differently than what we discussed in #736. Perhaps:

```suggestion
Starting from `aiida-quantumespresso==4.0`, the last three minor versions of Quantum ESPRESSO are supported. Older versions are supported up to a maximum of two years.
```

Reply:
I may have misunderstood what we discussed then. I thought even the last three minor versions would be subject to the 2 years max rule. For example, even if the oldest supported minor version happens to be 10 years old, do we still support it? Maybe it doesn't matter. And with what I propose, I guess we could technically end up with _no_ supported versions, if the most recent version is older than two years :sweat_smile: but in that case there might be bigger problems at hand.